README.md

```diff
@@ -36,12 +36,9 @@ Note! ZiGate open source ZigBee adapter hardware requires ZiGate firmware 3.1a o
 - To configure __pizigate__ port, specify the port, example : `/dev/serial0` or `/dev/ttyAMA0`
 - To configure __wifi__ ZiGate, manually specify IP address and port, example : `socket://192.168.1.10:9999` 
 
-__pizigate__ may requiert some adjustements on Rpi3 or Rpi4:
-- [Rpi3](https://zigate.fr/documentation/compatibilite-raspberry-pi-3-et-zero-w/)
-- [Rpi4](https://zigate.fr/documentation/compatibilite-raspberry-pi-4-b/)
-
-Note! Requires ZiGate firmware 3.1d and later
-- https://zigate.fr/tag/firmware/
+__pizigate__ does require ZiGate 3.1d firmware or later as well as some additional adjustements on Raspberry Pi 3/Zero, and 4:
```

Comment:
See previous comment
Firmware 3.1d it not recommended only for PiZiGate but for any ZiGate model

Reply:
Like so?